plugins/modules/azure_rm_keyvault.py

```diff
@@ -273,6 +273,13 @@
                         description:
                             - Property to specify whether NRP will ignore the check if parent subnet has serviceEndpoints configured.
                         type: bool
+    is_purge_deleted:
+        description:
+            - Whether permanently deletes the specified vault. aka Purges the deleted Azure key vault
+            - Default value is C(True)
```

Comment:
Can we change this to `False`? Otherwise, it is a breaking change?

Reply:
Of course, I will update it!